modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/reconstructedcell/v0.1.0.json

```diff
@@ -0,0 +1,158 @@
+      "bbplocsh": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/reconstructedcell/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/entity/v0.1.0",
+    "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id" : "this:ReconstructedCellShape",
+      "@type" : "sh:NodeShape",
+      "label" : "Reconstructed cell shape definition",
+      "targetClass" : "nsg:ReconstructedCell",
+      "nodeKind" : "sh:BlankNodeOrIRI",
+      "and":[
+        {
+          "node":"bbpentsh:EntityShape"
+        },
+        {
+          "property" : [
+            {
+              "path" : "nsg:projectName",
+              "name" : "Project name",
+              "description" : "Project name as given by the organization who generates the data",
+              "datatype": "xsd:string",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:axonProjection",
+              "name" : "Axon projection",
+              "description" : "Axon projection of the reconstructed cell.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:brainRegion",
+              "name" : "Brain region",
+              "description" : "Brain region where the reconstructed cell is located.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "minCount" : 1,
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:distanceToBoundary",
+              "name" : "Distance to boundary",
+              "description" : "Distance of the reconstructed cell to pia or to another brain boundary.",
+              "node" : "bbplocsh:DistanceToBoundaryShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:mType",
+              "name" : "Morphology type",
+              "description" : "The morphology type of the reconstructed cell: need the cell type for class restriction.",
+              "node": "bbptlotsh:CellTypeOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:mSubType",
+              "name" : "Morphology sub-type",
+              "description" : "The morphology sub-type of the reconstructed cell: need the cell sub-type for class restriction.",
+              "datatype": "xsd:string",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:subRegion",
+              "name" : "Sub region",
+              "description" : "Sub region where the reconstructed cell is located, CA1a in hippocampus for example.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:positionInLayer",
```

Comment:
The value of "nsg:positionInLayer" is not conform to [bbplocsh:BrainLocationShape](https://github.com/BlueBrain/nexus-bbp-domains/blob/master/modules/bbp-core/src/main/resources/schemas/bbp/core/brainLocation/v0.1.0.json).
I would suggest to have a property "nsg:brainLocation" defined as follows:

```json
{
              "path" : "nsg:brainLocation",
               "name" : "Brain location",
               "description" : "Location (brain region, position In Layer, longitudinalAxis) of the reconstructed cell.",
              "node":"bbplocsh:BrainLocationShape",
               "maxCount" : 1
}

Reply:
The solution above will allow to merge in one property all information to locate the reconstructed cell:

* positionInLayer
* longitudinalAxis
* layer

Reply:
OK for positionInLayer and longitudinalAxis.
Layer is a field that they keep apart from the brain region and that is not as such in the parcellation schemes or Uberon. suggest to keep it apart.